aiida_mlip/helpers/help_load.py

```diff
@@ -38,11 +44,13 @@ def load_model(model: Optional[Union[str, Path]], architecture: str) -> ModelDat
     elif (file_path := Path(model)).is_file():
         loaded_model = ModelData.local_file(file_path, architecture=architecture)
     else:
-        loaded_model = ModelData.download(model, architecture=architecture)
+        loaded_model = ModelData.download(
+            model, architecture=architecture, cache_dir=cache_dir
+        )
     return loaded_model
 
 
-def load_structure(struct: Optional[Union[str, Path, int]]) -> StructureData:
+def load_structure(struct: Union[str, Path, int] = None) -> StructureData:
```

Comment:
thing is, when I don't pass anything and I call it like
```
struct = load_structure()
```
it will consider that struct does not exist and won't work, so I now just kept both Optional and the default value None. If it makes sense (there might be a better solution)

Reply:
`Optional` just means it accepts a `Nonetype`, so you can have either
```suggestion
def load_structure(struct: Optional[Union[str, Path, int]] = None) -> StructureData:
```
or
```suggestion
def load_structure(struct: Union[str, Path, int, None] = None) -> StructureData:
```